ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/workers-marketplace/byo-subnets/cucushift-installer-rehearse-aws-ipi-workers-marketplace-byo-subnets-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-workers-marketplace-byo-subnets
```

Comment:
Is this workflow used in the jobs?

Reply:
I was planing to use this install workflow to cover marketplace ci jobs, in the last minutes, I decided to do that later, just developed the new install workflow.